src/concepts/censuses.md

```diff
@@ -1,14 +1,15 @@
-## Censuses
+## Legacy Census Metrics
```

Comment:
Should this section also include the definitions for aDAU and aMAU?

Reply:
fwiw there's some internal commentary at https://mozilla-private.report/smoot-existing-metrics/book/20_metric_correlations.html#dau-and-dau-5-uris; it's probably good to have an aDAU grave marker on DTMO and I'll toss that on my todo list.

Reply:
Yeah, moving https://docs.telemetry.mozilla.org/cookbooks/active_dau.html to the historical section is also on my list of things to do (of course I wouldn't object if @tdsmith did it for me!)